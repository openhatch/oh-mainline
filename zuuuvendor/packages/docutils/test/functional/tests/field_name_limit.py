# Source and destination file names.
test_source = "field_list.txt"
test_destination = "field_name_limit.html"

# Keyword parameters passed to publish_file.
reader_name = "standalone"
parser_name = "rst"
writer_name = "html"

# Settings
settings_overrides['field_name_limit'] = 0 # no limit
settings_overrides['docinfo_xform'] = False
# local copy of default stylesheet:
settings_overrides['stylesheet_path'] = ( 
    'functional/input/data/html4css1.css')
