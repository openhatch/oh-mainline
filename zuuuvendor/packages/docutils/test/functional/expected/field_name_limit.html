<?xml version="1.0" encoding="utf-8" ?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="Docutils 0.11: http://docutils.sourceforge.net/" />
<title></title>
<link rel="stylesheet" href="../input/data/html4css1.css" type="text/css" />
</head>
<body>
<div class="document">


<table class="docutils field-list" frame="void" rules="none">
<col class="field-name" />
<col class="field-body" />
<tbody valign="top">
<tr class="field"><th class="field-name">short:</th><td class="field-body">This field's name is short.</td>
</tr>
<tr class="field"><th class="field-name">medium-length:</th><td class="field-body">This field's name is medium-length.</td>
</tr>
<tr class="field"><th class="field-name">long field name:</th><td class="field-body">This field's name is long.</td>
</tr>
<tr class="field"><th class="field-name">very very long field name:</th><td class="field-body">This field's name is quite long.</td>
</tr>
</tbody>
</table>
</div>
</body>
</html>
